src/components/App.js

```diff
@@ -52,11 +52,11 @@ const App = () => {
   }
 
   const handleSelectFilterChange = (event) => {
-    setFilterTodos(event.target.value)
+    setStatusFilter(event.target.value)
   }
 
-  const filterTodosStatus = (todo) => {
-    return filterTodos === 'all' ? true : todo.status === filterTodos
+  const statusFilterStatus = (todo) => {
```

Comment:
esta es una función, por lo que buena práctica de nombre debería comenzar con un verbo

```suggestion
  const filterTodoByStatus = (todo) => {
```